tests/wpunit/Tribe/Tickets/Commerce/PayPal/Custom_ArgumentTest.php

```diff
@@ -68,7 +68,6 @@ public function should_allow_getting_the_original_custom_arguments() {
 		$encoded = urlencode_deep( json_encode( $args ) );
 
 		$this->assertEquals( (object) $args, Custom_Argument::decode( $encoded ) );
-		$this->assertEquals( $args, Custom_Argument::decode( $encoded, true ) );
```

Comment:
Why this one is removed?

Reply:
Actually, both have to be removed. Because the `decode` method has changed, given that paypal changed the way they return their data.

@lucatume can we get rid of those two assertions?

Reply:
I think we are ok on removing, since the other tests cover the bases, and the changes on the code above seems to be invalidating this test.